app/Http/Controllers/Task/UserController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Http\Controllers\Task;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use App\Models\User;
+use App\Http\Requests\UsersRequest;
+
+class UserController extends Controller
+{
+    public function __construct(User $modelUser)
+    {
+        $this->modelUser = $modelUser;
+    }
+
+    public function index()
+    {
+        return view('auth.auth.register');
+    }
+
+    public function store(UsersRequest $request)
+    {
+        $userArr = [
+            'username' => $request->username,
+            'password' => bcrypt($request->password),
```

Comment:
nghiên cứu sử dụng mutator: https://laravel.com/docs/5.6/eloquent-mutators#accessors-and-mutators